charts/newrelic-infrastructure/templates/controlplane/daemonset.yaml

```diff
@@ -72,14 +85,11 @@ spec:
                 fieldRef:
                   apiVersion: "v1"
                   fieldPath: "status.hostIP"
-            {{- if .Values.verboseLog }}
-            - name: NRI_KUBERNETES_VERBOSE
-              value: "1"
-            {{- end }}
-            {{- with .Values.controlPlane.extraEnv }}
+
+            {{- with .Values.controlPlane.env }}
             {{- toYaml . | nindent 12 }}
             {{- end }}
-          {{- with .Values.controlPlane.extraEnvFrom }}
+          {{- with .Values.controlPlane.envFrom }}
```

Comment:
I'm not a big fan of removing the `extra` prefix. Do we have a reason for doing that? Users might get the impression they are fully overriding the `env` section of the pod, while that's not the case: We're appending.

Reply:
Paolo stated this [above](https://github.com/newrelic/nri-kubernetes/pull/383#discussion_r845194876) and this was solved in [#64e304d](https://github.com/newrelic/nri-kubernetes/pull/383/commits/64e304d72084e4070dc15553499680bc4f1ba85a).